### PR TITLE
Update github-git and github-bash cheatsheets

### PR DESCRIPTION
**Pull request by NishkarshRaj**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>
Updated the Git cheatsheet where the git init method was explained. I added the fact that after creating the local repository, we have to link it with the remote repository using some link name like origin.
<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>
@brianamarie Please review 🙂

### CODE REVIEW

**Comment by NishkarshRaj** [comment]
@brianamarie I have explained the permissions used in Linux Administration in the second commit.

File: downloads/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
 
 ```$ git init```
 
+After using the init command, the local repository must be linked to the GitHub repository created on our account using the following command:
+
+```$ git remote add origin "https://www.github.com/<<username>>/<<Repository name>>.git"```
```
Comment (brianamarie) [inline downloads/github-git-cheat-sheet.md]:
```suggestion
```$ git remote add origin [url]```
```

This slight change would keep consistent syntax between this command and the `git clone` command.

File: downloads/github-git-cheat-sheet.md [hunk 2]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
 
 ```$ git init```
 
+After using the init command, the local repository must be linked to the GitHub repository created on our account using the following command:
```
Comment (brianamarie) [inline downloads/github-git-cheat-sheet.md]:
```suggestion
After using the `git init` command, link the local repository to an empty GitHub repository using the following command:
```

File: downloads/github-bash-cheat-sheet.md [hunk 3]
```diff
@@ -148,6 +148,7 @@ Change read, write, and execute permissions on files and folders
 
 Change permissions of [file] to 755
 
+> Octal representation of permissions are group of permissions for User (u), Group (g) and Others(o) with values that are sum of read (4), write (2) and execute (1) permissions.
```
Comment (brianamarie) [inline downloads/github-bash-cheat-sheet.md]:
```suggestion
> Octal representation of permissions are group of permissions for User (u), Group (g) and Others (o) with values that are sum of read (4), write (2) and execute (1) permissions. For example, 755 is:
> - Owner = 7; read (4) + write (2) + execute (1)
> - Group = 5; read (4) + execute (1)
> - Others = 5; read (4) + execute (1)


```

**Comment by NishkarshRaj** [comment]
Thanks alot @brianamarie for helping me out. 👍 
I have committed all of your suggestions that were to the point and generalized.

**Comment by brianamarie** [comment]
Great. Thank you again for your contributions, @NishkarshRaj! :octocat: 🙇

**Comment by NishkarshRaj** [comment]
Thankyou @brianamarie , this is my first ever contribution on any GitHub organization project. ❤️

File: downloads/github-git-cheat-sheet.md [hunk 4]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:64]: @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 5]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
 
 ```$ git init```
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:65]: @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 6]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
 
+After using the init command, the local repository must be linked to the GitHub repository created on our account using the following command:
+
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:68]: @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 7]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
+```$ git remote add origin "https://www.github.com/<<username>>/<<Repository name>>.git"```
+
 Turn an existing directory into a Git repository
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:71]: @ @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 8]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:72]: @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 9]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
 
 ```$ git clone [url]```
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:73]: @brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 10]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
 
 ```$ git init```
 
+After using the `git init` command, link the local repository to an empty GitHub repository using the following command:
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:67]: git init

Comment (Salvador1313) [reply downloads/github-git-cheat-sheet.md:67]: Ковычки забыл

File: downloads/github-git-cheat-sheet.md (hunk 6 above)
Comment (Salvador1313) [reply downloads/github-git-cheat-sheet.md:68]: https://github.com/brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 11]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
+
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:68]: https://github.com/brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 12]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
+```$ git remote add origin [url]```
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:69]: https://github.com/brianamarie

Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:69]: https://github.com/brianamarie

File: downloads/github-git-cheat-sheet.md [hunk 13]
```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
...
+
```
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:70]: https://github.com/brianamarie

File: downloads/github-git-cheat-sheet.md (hunk 11 above)
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:68]: GitHub,

File: downloads/github-git-cheat-sheet.md (hunk 12 above)
Comment (Salvador1313) [inline downloads/github-git-cheat-sheet.md:69]: https://github.com/brianamarie

File: downloads/github-git-cheat-sheet.md (hunk 11 above)
Comment (Salvador1313) [reply downloads/github-git-cheat-sheet.md:68]: Голова не рубит совсем уже. Сори

**Comment by Salvador1313** [comment]
+ "get init``  https://github.com/brianamarie
+
+` '$ git [GitHub]``

**Comment by Salvador1313** [comment]
Спасибо. Но клавиатуру надо нову а то без кнопок не очень

File: downloads/github-git-cheat-sheet.md (hunk 1 above)
Comment (Salvador1313) [reply downloads/github-git-cheat-sheet.md]: -` '$ git origin  "https://www.github.com/<<Salvador1313><Salvador1313> /<Salvador1313>><<https://github.com/brianamarie><https://github.com/brianamarie>.git<https://github.com/brianamarie>>"``

File: downloads/github-git-cheat-sheet.md (hunk 2 above)
Comment (Salvador1313) [reply downloads/github-git-cheat-sheet.md]: - init` $ <<https://github.com/Salvador1313>><https://github.com/brianamarie>>`''

File: downloads/github-bash-cheat-sheet.md (hunk 3 above)
Comment (Salvador1313) [reply downloads/github-bash-cheat-sheet.md]: (1). - @brianamarie. = >7; Salvador1313 (4)  + github.com  (2)  +  755 (1)

Здаюсь...
